.github/actions/setup-flutter/action.yml

```diff
@@ -0,0 +1,18 @@
+name: Setup 
+description: Setup runner to be able to build an app with Flutter. Not specific for Android or iOS.
+
+runs:
+  using: "composite"
+  steps:
+  # Install latest stable version of Flutter. Mostly because k-paxian/dart-package-publisher always uses the latest stable version of Flutter and we want the rest of the CI to match that. 
```

Comment:
Do you think it can break something in future if there is major release for Flutter and we haven't tested in with our SDK?

Reply:
That is a good point, but it also make sure to keep us updated with the latest version and if our SDK doesn't break with it. We can revisit it and hardcode the version here if needed.

Reply:
I thought of this too, but I also noticed that `k-paxian/dart-package-publisher` doesn't let you define a flutter version. We used to run tests/lint with flutter version X but deploy (using `k-paxian/dart-package-publisher`) with flutter version Y and it caused deployments to fail. 

Therefore, no matter what we do, let's make sure to always use the same version of Flutter for the whole CI pipeline. At this time, because we can't specify flutter version during deployment, is the latest stable version.